### ISSUE TITLE
package testing with xvfb-run

### ISSUE THREAD

**Issue by sodface** [Tomas-M/xlunch#139]
Hello, I'm working on a package for Alpine Linux. I can use Alpine's option to skip tests but I see that many graphical packages in Alpine use xvfb-run for headless tests, eg `xvfb-run make test` unfortunately this hangs forever on the build server. I assume waiting on user input that never comes?  

`xvfb-run make livetest` works though I believe the `--scroll` argument should be `--noscroll`?

```
./xlunch -g extra/wp.jpg -f "extra/OpenSans-Regular.ttf/10" -i /tmp/xlunch-tmp.dsv -b 140 --iconpadding 20 --textpadding 10 --paddingswap --leastmargin 10 --scroll || true
./xlunch: option is ambiguous: scroll
See --help for usage documentation
```
Is `make livetest` a suitable alternative to `make test` or should I just disable testing altogether?

//edit, I think the only reason `xvb-run make livetest` doesn't hang is because xlunch doesn't actually run due to the ambiguous scroll option. I'm not very familiar with headless testing with xvfb-run but it seems like skipping the test is the right answer here.

**Comment by PMunch** [comment]
I believe the testing in `make test` and `make livetest` is more there to have a faster command to run while developing and manually testing the application. I believe there is no actual tests written for xlunch.

But why did you close this issue?

**Comment by sodface** [comment]
> But why did you close this issue?

I use xlunch in a touch screen project which I'm trying to finish up after some months of inactivity. I updated my local Alpine package to the latest xlunch release and I've been spending some time here in the repository reviewing issue comments related to transparency.

I kept seeing this open issue that I submitted and since 6 months had passed without a reply, I interpreted that to mean, perhaps incorrectly, as "won't fix". And based on your comment above, it seems I misinterpreted the purpose of `make test` anyway so I don't think there's anything to "fix", other than perhaps the `--scroll` option which is a separate issue. Additionally, I'd decided against submitting a package to Alpine so for my local builds this is a non-issue.

**Comment by PMunch** [comment]
Aah I see.

Cool that you're doing a touch screen project. That's exactly the thing which brought me to xlunch in the first place.

**Comment by sodface** [comment]
xlunch is perfect for a touch screen application. Besides a little struggling with the config initially, it's working great. I'm using it in desktop mode and keeping one workspace unoccupied so that when a two finger tap in a screen hotspot is detected, I just jump to that workspace, which gives the impression that the menu has appeared. Visually, it works better than displaying and then dismissing an xlunch floating menu.

Also, when I said "latest xlunch release" I meant latest tag, as the actual releases are a bit behind the tagged version.
